### PR TITLE
CoeursVaillants - Correction lien image

### PR DESCRIPTION
**Pull request by MJKoshon**
## Changes / Comments

correction lien image

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [x] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

File: CoeursVaillants/CoeursVaillants.html [hunk 1]
```diff
@@ -7,8 +7,7 @@
         <label>Peuple <input type="text" name="attr_peuple"> Poids <input type="text" name="attr_poids"></label>
     </div>
     <div class="sheet-logo sheet-block">
-<!        <img class="sheet-logo" src="https://cdn.discordapp.com/attachments/896513056824234054/896513122121171004/unknown.png" width="100%" />!>
-        <img class="sheet-logo" src="https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/CoeursVaillants/images/logo.png" width="100%" />
+        <img class="sheet-logo" src="https://cdn.discordapp.com/attachments/896513056824234054/896513122121171004/unknown.png" width="100%" />
```
Comment (Stexinator) [inline CoeursVaillants/CoeursVaillants.html:10]: You should have the images in the roll20 repo. Else they might be deleted and noone is around to fix it

Comment (MJKoshon) [reply CoeursVaillants/CoeursVaillants.html]: I push a new one, think this is good this time. >> Fix Logo (d61a397)

title="Version 1.00 - 10/12/2021" src="https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/CoeursVaillants/images/Logo.png"

**Comment by nmbradley** [comment]
@MJKoshon Could you please indicate the name of the sheet in the title of the pull request? It makes it much easier for us to resolve issues in this repository if PRs and Issues are accurately labeled.

**Comment by MJKoshon** [comment]
Done

**Pull request merged**